docs/docs/deploying-to-now.md

```diff
@@ -2,8 +2,20 @@
 title: Deploying to Now
 ---
 
-In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can do the following:
+In order to deploy your Gatsby project using [Now](https://zeit.co/now), depending on how big your project is, you can do the following:
 
+### If your site has few pages
+1.  Install the Now CLI
+
+`npm install -g now`
+
+2.  Run `gatsby build` at the root of your Gatsby project
+
+3. Then run `now ./public` at the root of your Gatsby project
+
+Simply, you can write a npm script `deploy: "gatsby build && now ./public"`
+
+### If your project size is big and includes many pages
```

Comment:
Can you explain the difference here? What's the benefit of using a serve step vs. just deploying the whole builder?

Reply:
It's from #8943 . Build locally then upload only public files can make it faster and not error-prone. It's practically so-called **static deploy**. It''s really suit for small or middle sites.